### PR TITLE
Fixes for networkprinters that are not part of Cura Connect

### PR DESCRIPTION
**Pull request by fieldOfView**
This PR fixes how connected networkprinters that are not part of Cura Connect are handled in the interface. Network printers that are using eg OctoPrint (but also others, such as Repetier) don't have "connect_group_name"  metadata and may not have the concept of configurations. The PR also fixes a repeating QML warning for every non-networked printer that is added to Cura (including UM2+).

### CODE REVIEW

**Comment by ChrisTerBeke** [comment]
Please also check the cloud printing PR, this work overlaps some of that (and might make your fixes easier to implement?)

**Comment by fieldOfView** [comment]
I had a quick look. The fixes in this PR don't seem to conflict with the cloud work, which to me suggest that the issues that the PR fixes are not fixed in your PR.

**Comment by ChrisTerBeke** [comment]
Well, at least they touch the same lines of code, so just a heads up to prevent conflicts.

**Comment by diegopradogesto** [comment]
Devs. CURA-6098

File: resources/qml/PrinterSelector/MachineSelector.qml [hunk 1]
```diff
@@ -26,7 +26,14 @@ Cura.ExpandablePopup
 
     headerItem: Cura.IconWithText
     {
-        text: isNetworkPrinter ? Cura.MachineManager.activeMachineNetworkGroupName : Cura.MachineManager.activeMachineName
+        text:
+        {
+            if (isNetworkPrinter && Cura.MachineManager.activeMachineNetworkGroupName != "")
+            {
+                Cura.MachineManager.activeMachineNetworkGroupName
```
Comment (nallath) [inline resources/qml/PrinterSelector/MachineSelector.qml]: You missed a return here. Fixed that for you in b1b61fa466eafa51121ce7a276dc2046d5f38ea7

**Pull request merged**